packages/pricing/src/services/pricing-module.ts

```diff
@@ -248,12 +281,247 @@ export default class PricingModuleService<
     ]
   }
 
+  @InjectManager("baseRepository_")
```

Comment:
todo: l221 should we use serialize instead?

Reply:
I tried that already, it explodes if its not serializing mikroorm entities.